src/layout/Footer.tsx

```diff
@@ -25,180 +26,187 @@ export const Footer = () => {
   return (
     <Box
       component="footer"
-      sx={{
-        padding: '1rem',
-        bgcolor: 'info.light',
-        display: 'grid',
-        gridTemplateColumns: { xs: '1fr', md: '1fr 1fr 1fr' },
-        gap: '3rem',
-        color: 'primary.main',
-      }}>
-      <div>
-        <Box sx={{ display: 'flex', gap: '0.5rem', justifyContent: 'space-between' }}>
-          <Typography color="primary" sx={{ fontWeight: 'bold' }}>
-            {t('footer.about_nva')}
+      sx={{ display: 'flex', flexDirection: 'column', padding: '1rem', bgcolor: 'info.light', color: 'primary.main' }}>
+      <Button
+        sx={{ width: 'fit-content', alignSelf: 'center' }}
```

Comment:
Flex kan (og burde kanskje helst) stå for hvor elementene skal legge seg selv når man kan, istedet for å override per element

```suggestion
      sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center', padding: '1rem', bgcolor: 'info.light', color: 'primary.main' }}>
      <Button
```